libraries/Microsoft.Bot.Connector/Authentication/JwtTokenValidation.cs

```diff
@@ -75,6 +75,17 @@ public static async Task<ClaimsIdentity> ValidateAuthHeader(string authHeader, I
             {
                 return await EmulatorValidation.AuthenticateEmulatorToken(authHeader, credentials, httpClient ?? _httpClient, channelId);
             }
+            else if(channelProvider != null && !string.IsNullOrEmpty(await channelProvider.GetChannelServiceAsync().ConfigureAwait(false)))
+            {
```

Comment:
This line is too complex. Please simplify. Strongly consider puting the async call on it's own line, not direclty in the if statement.

Reply:
Clean as a whistle now.